diff --git a/conda/core/solve.py b/conda/core/solve.py
index 328b98f..0e7e2d4 100644
--- a/conda/core/solve.py
+++ b/conda/core/solve.py
@@ -1229,7 +1229,8 @@ class Solver:
         return ssc
 
     def _notify_conda_outdated(self, link_precs):
-        if not context.notify_outdated_conda or context.quiet:
+        import sys
+        if sys.platform == "emscripten" or not context.notify_outdated_conda or context.quiet:
             return
         current_conda_prefix_rec = PrefixData(context.conda_prefix).get("conda", None)
         if current_conda_prefix_rec:
